specs/interop/predeploys.md

```diff
@@ -430,6 +442,29 @@ dependency set called `dependencySet()`. This function MUST return the array of
 `L1Block` MUST also provide a public getter to get the dependency set size called `dependencySetSize()`. This function
 MUST return the length of the dependency set array.
 
+### Deposit Context
+
+The `L1Block` contract is updated with new methods to interact with [deposit contexts](./derivation.md#deposit-context).
+
+```solidity
+function isDeposit() public view returns (bool);
+function resetIsDeposit() public;
```

Comment:
The function name is `resetIsDeposit` here but `depositsComplete` elsewhere - what is the final name?

Reply:
good catch! thanks :)